templates/packer/aws/author-publish-dispatcher.json

```diff
@@ -4,6 +4,7 @@
     "aws_subnet_id": "",
     "root_volume_size": "20",
     "tmp_dir": "/tmp/shinesolutions/packer-aem",
+    "inspec_dir": "/opt/puppetlabs/puppet/bin",
```

Comment:
Please rename `inspec_dir` to `puppet_bin_dir` because this is not strictly for inspec. All ruby gems with binary commands will be available on this path when they're provisioned via puppet gem.

Reply:
But there is an existing user variable with the same name  "puppet_bin_dir": "/opt/puppetlabs/bin"?

Reply:
Renamed to `rubygems_bin_dir` .